catalog/core/catalog-core-api-impl/src/test/java/ddf/catalog/data/impl/MetacardTypeImplTest.java

```diff
@@ -123,6 +129,27 @@ public void testEquals() {
     }
 
     @Test
+    public void testHashCode() {
+        MetacardTypeImpl metacardType1 = generateMetacardType("test", 0);
```

Comment:
Unless we're asserting on making sure that metacard type contains that exact string i would almost say there's no need. They are only tests and refactoring things too much can make it just as unreadable. These are simple strings and not used in the assert and see no reason to make a constant unless the consistency of that string is affecting the test output in some way.